src/js/index.js

```diff
@@ -7,3 +7,24 @@ navigation()
 bookmark()
 formular()
 quizCardAnswer()
+
+// IDEA: randomizer button for random question
+
+const quizCard = document.querySelector('[data-js=quiz-card]')
+const buttonRandom = document.querySelector('[data-js=randomButton]')
+const number = quizCard.length
+
+buttonRandom.addEventListener('click', () => {
+  const questions = document.querySelectorAll('[data-js="quiz-card"]')
+  const randomIndex = getRandom(number - 1)
+  questions.forEach((question, index) => {
```

Comment:
Now this also needs to change to quiz card
```suggestion
  quizCards.forEach((question, index) => {
```